tests/_utils/check_sync_diff

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# parameter 1: config file for sync_diff_inspector
+# parameter 2: max check times
+
+conf=$1
+if [ $# -ge 2 ]; then
+    check_time=$2
+else
+    check_time=10
+fi
+
+LOG=$DUMPLING_OUTPUT_DIR/sync_diff_inspector.log
+
+i=0
+while [ $i -lt $check_time ]
+do
+    bin/sync_diff_inspector --config=$conf >> $LOG 2>&1
+    ret=$?
+    if [ "$ret" == 0 ]; then
```

Comment:
Perhaps better use `set -e` at the beginning...

Reply:
I didn't get it, `set -e` means it will exit immediately, but this shell need retry